pkgs/tools/security/nitrokey-app2/pyqt5-stubs.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pyqt5
+, pyqtwebengine
+, pyqtchart
+, pyqt3d
+, pyqtdatavisualization
+, pytestCheckHook
+, pytest-xvfb
+}:
+
+buildPythonPackage rec {
+  pname = "pyqt5-stubs";
+  version = "5.15.6.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "python-qt-tools";
+    repo = "PyQt5-stubs";
```

Comment:
This is not specific to nitrokey, right? So we should move it to python-modules/pythonPackages